modules/velocloud-sdwan/README.md

```diff
@@ -0,0 +1,92 @@
+| <a name="input_platform"></a> [platform](#input\_platform)                                                                                                                                                                                                                                            | Device platform flavor that determines number of CPU cores and memory                                                                                                                                                                                                                                                                                                                                                                            | `string`                                                                      | n/a                                     |   yes    |
+| <a name="input_project_id"></a> [project\_id](#input\_project\_id)                                                                                                                                                                                                                                    | project\_id                                                                                                                                                                                                                                                                                                                                                                                                                                      | `string`                                                                      | n/a                                     |   yes    |
+| <a name="input_package_code"></a> [package\_code](#input\_package\_code) Use [equinix_network_device_software](https://registry.terraform.io/providers/equinix/equinix/latest/docs/data-sources/equinix_network_device_software) datasource with device type code to find the supported package codes | Device software package                                                                                                                                                                                                                                                                                                                                                                                                                          | `string`                                                                      | n/a                                     |   yes    |
```

Comment:
Could you put the reference link in the description section?

Reply:
fixed